development/mimir-microservices-mode/docker-compose.jsonnet

```diff
@@ -62,12 +64,14 @@ std.manifestYamlDoc({
       name: 'distributor-1',
       target: 'distributor',
       httpPort: 8000,
+      extraArguments: '-distributor.ha-tracker.consul.hostname=consul:8500',
```

Comment:
This seems like it should be conditional based on the `enable_ha_tracker` config as well as the other HA config bits in `mimir.yaml`.

Reply:
Or alternatively, the jsonnet config should be removed so HA is always used.

Reply:
I went with option 2, always have HA used and group the 2 new prometheus with existing `enable_prometheus` toggle